sechub-solution/helm/sechub-server/values.yaml

```diff
@@ -0,0 +1,271 @@
+# templateMetadataAnnotations: |-
+#   prometheus.io/scrape: "true"
+#   prometheus.io/probe: "true"
+#   prometheus.io/port: "10250"
+#   prometheus.io/path: "/actuator/prometheus"
+
+storage:
+  local:
+    # No shared storage - only for standalone SecHub server (stores into local /tmp)
+    enabled: true
+  s3:
+    # Use S3 as shared storage backend
+    enabled: false
+    endpoint: "https://to.be.defined:<port>"
+    bucketname: "<my-bucketname>"
+    # Defines the access key for used S3 bucket
+    # Put the S3 access key in k8s secret secret-storage:sechub_storage_s3_accesskey
+    # Put the S3 secret key in k8s secret secret-storage:sechub_storage_s3_secretkey
+  sharedVolume:
+    # Use a shared filesystem as storage backend
+    enabled: false
+    dir: "/nfs/sechub-share"
+
+volumes:
+  tmpVolume:
+    # Size of /tmp in container
+    size: 10Gi
+
+webScan:
+  # Strategy to distinguish intranet/internet targets for DAST/ web scans
+  intranetDetection:
+    # Example: Intranet host names end with `.local` or with `.intranet.example.org`. E.g. https://myhost.local
+    uriPattern: "intranet-hostname-ends-with:.local,.intranet.example.org"
+    # Example: Intranet host's ipv4 range is 192.168.0.0/16 or ipv6 range 2001:db8:85a3:0:0:8a2e:370:*
+    ipPattern: "intranet-ip-pattern:192.168.*.*,[2001:db8:85a3:0:0:8a2e:370:*]"
+
+# Set user id and group id of the application user
+runAs:
+  uid: 9002
+  gid: 9002
+
+# deploymentComment (optional):
+#  When setting to a different value than before, it forces k8s to spin up a new container.
+#  This way, you can force deployments e.g. when only secrets have changed.
+deploymentComment: "my deployment comment"
+
+# ----------------------------------------------------------------------------------------------#
+# Settings for Scan Backends (at least one should be enabled)
+#   For activation: set enabled: true
+# --------------------------------------------------------------------------------------------- #
+
+# ------------------------------------------------------------------------------------- #
+# Checkmarx adapter (Deprecation warning: this adapter is not being developed further)
+# ------------------------------------------------------------------------------------- #
+adapterCheckmarx:
+  enabled: false
+  # Base url of your Checkmarx server
+  baseURL: https://to.be.defined
+  # Put the user id  of Checkmarx user in k8s secret secret-checkmarx:sechub_adapter_checkmarx_userid
+  # Put the password of Checkmarx user in k8s secret secret-checkmarx:sechub_adapter_checkmarx_password
+  #
+  # Interval in minutes when SecHub checks if the scan has finished on Checkmarx side
+  checkIntervalInMinutes: 1
+  # Time in minutes when adapter result check will be canceled/time out. When -1 timeout is 7200 minutes
+  timeoutInMinutes: 600
+  # Team ID of a newly created Checkmarx project.
+  # For project creation, a team must be defined.
```

Comment:
```suggestion
  # For project creation, a Checkmarx team id must be defined.
```

Reply:
Fixed. (The preceding comment line contained basically the same informations)